packages/cozy-harvest-lib/src/components/TriggerLauncher.jsx

```diff
@@ -82,24 +107,35 @@ export class TriggerLauncher extends Component {
   }
 
   async handleError(error) {
+    /**
+     * We don't setState error / running here, since it's the job of `refeshTrigger`
```

Comment:
This is also true for handleSuccess etc. i think it may be nest to put that next to handle Update.